### DISCUSSION TITLE
How to change the "focus ring" to a dotted line instead of this atrocity

### DISCUSSION THREAD

**Discussion by TheQwertyOne** [black7375/Firefox-UI-Fix#569] [Q&A]
Hello.
Focus Ring is a UI element that surrounds focused items (links, buttons etc.) with a visible frame when switching between them using Tab. It also often highlights whatever is being activated by the mouse click. Normally it looks like an unintrusive dotted line.
After installing the UI conversion, I got this unwanted change. How can I get it back to a dotted line without it being overwritten with futher updates? 
![firefox_or2nAYVIqD](https://user-images.githubusercontent.com/100481618/211130393-bd3a50aa-b1a4-4051-8816-6f8a200eb167.png)

I had already tried changing _about:config_ (style and width) settings, they do nothing apart from disabling it entirely.
I don't want to use an addon, like Stylus, to add another layer of change (and computation) on top of it.

Thank you for your attention.

**Comment by black7375** [comment]
I tried to inactive the theme, but the dotted line does not appear.
Is there a way to reproduce?
![image](https://user-images.githubusercontent.com/25581533/211160771-222e5c7d-4a0e-4d1f-8cf1-8f11c5b9f1f1.png)
